packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -253,7 +259,7 @@ const FrontpageReviewWidget = ({classes, showFrontpageItems=true}: {classes: Cla
     <div>
       <SectionTitle 
         title={<LWTooltip title={overviewTooltip} placement="bottom-start">
-          <Link to={reviewPostPath || ""}>
+          <Link to={"/reviewVoting"}>
```

Comment:
@jpaddison3 might have an opinion on this change?

Reply:
I believe we now don't use that variable, but otherwise seems like a reasonable change. I might have a bias towards changing mid-stream, but I think on balance I agree with the change.

Reply:
I'm not sure I understood JP's point here, but this seems easily reversible if we're like "nah that was the wrong call"

Reply:
`s/bias towards/bias against/`